pkg/storage/stores/shipper/compactor/deletion/mode_test.go

```diff
@@ -31,3 +27,21 @@ func TestParseMode(t *testing.T) {
 	_, err = ParseMode("something-else")
 	require.ErrorIs(t, errUnknownMode, err)
 }
+
+func TestFilteringEnabled(t *testing.T) {
+	enabled, err := FilteringEnabled("disabled")
```

Comment:
This is the method we can change because `DeleteEnabled` now implies filtering

Reply:
Changed to use a new method on the mode struct.